### PR TITLE
(Update) Build and Push

### PR DESCRIPTION
**Pull request by KinnonYee-harness**
Thanks for contributing to the Harness Developer Hub! Our code owners will review your submission.

## Description

* Please describe your changes: \__________  JPMC required additional clarifications in the documentation.
* Jira/GitHub Issue numbers (if any): \___________.  https://harnesssupport.zendesk.com/agent/tickets/73024
* Preview links/images (Internal contributors only): \__________________

## PR lifecycle

We aim to merge PRs within one week or less, but delays happen sometimes.

If your PR is open longer than two weeks without any human activity, please tag a [code owner](https://github.com/harness/developer-hub/blob/main/.github/CODEOWNERS) in a comment.

PRs must meet these requirements to be merged:

- [ ] Successful preview build.
- [ ] Code owner review.
- [ ] No merge conflicts.
- [ ] Release notes/new features docs: Feature/version released to at least one prod environment.

### CODE REVIEW

**Comment by KinnonYee-harness** [comment]
@dewan-ahmed 
Looks like it's good now.  Not sure where the error was.  I added a line break between this section and the rest, but I don't see this section in most other documentation, so not sure what it's for.
```
import Tabs from '@theme/Tabs';
import TabItem from '@theme/TabItem';
```

File: docs/continuous-integration/use-ci/build-and-upload-artifacts/build-and-push-nonroot.md [hunk 1]
```diff
@@ -7,13 +7,22 @@ sidebar_position: 23
...
+
+### Build X Builds
 Customers can also utilize BuildX for their builds.  This feature can be enabled by submitting a ticket to Harness Support to enable the flag `CI_USE_BUILDX_ON_K8`.  We recommend [reviewing the notes about this feature on the CI Early Access Features page](https://developer.harness.io/docs/continuous-integration/ci-supported-platforms/#harness-ci-early-access-features).  You must run BuildX on Kubernetes with Privileged mode enabled.
 
-Other build infrastructures use [drone-docker](https://github.com/drone-plugins/drone-docker/blob/master/README.md).
+### Drone Docker
+It is also possible to utilize drone docker. Please read the following documentation for a full breakdown of [drone-docker] and how to use it (https://github.com/drone-plugins/drone-docker/blob/master/README.md).
```
Comment (dewan-ahmed) [inline docs/continuous-integration/use-ci/build-and-upload-artifacts/build-and-push-nonroot.md:20]:
```suggestion
It is also possible to utilize drone docker. Please read the following documentation for a full breakdown of [drone-docker](https://github.com/drone-plugins/drone-docker/blob/master/README.md) and how to use it.
```

**Review by dewan-ahmed — COMMENTED** [review]
Fixing formatting of a link.

**Comment by KinnonYee-harness** [comment]
Hi @jamie-harness .  I spoke with Nofar, and she asked me to have you review this document change I've made

**Review by jamie-harness — APPROVED** [review]

**Review by nofarblue — APPROVED** [review]

**Pull request merged**